src/rpc/handlers/NFTInfo.cpp

```diff
@@ -119,37 +43,36 @@ doNFTInfo(Context const& context)
         return *status;
     auto const lgrInfo = std::get<ripple::LedgerInfo>(maybeLedgerInfo);
 
-    std::optional<Backend::NFT> dbResponse =
-        context.backend->fetchNFT(tokenID, lgrInfo.seq, context.yield);
+    std::optional<Backend::NFT> dbResponse;
+    try
+    {
+        dbResponse =
+            context.backend->fetchNFT(tokenID, lgrInfo.seq, context.yield);
+    }
+    catch (Backend::UnexpectedDataError const& e)
+    {
+        return Status{
+            RippledError::rpcDB_DESERIALIZATION,
+            "NFT has no URI recorded. Try again in a moment and this error "
+            "still persists open an issue as this is unexpected"};
```

Comment:
note reviewers - please look at this wording. i don't expect this error so i would legitimately want an issue opened. but not sure what you think in terms of verbiage.